src/schema/sources.ts

```diff
@@ -1043,6 +1042,11 @@ const membershipsPageGenerator = offsetPageGenerator<GQLSourceMember>(100, 500);
 
 const sourcePageGenerator = offsetPageGenerator<GQLSource>(100, 500);
 
+const categoriesPageGenerator = offsetPageGenerator<GQLSourceCategory>(
+  100,
+  500,
```

Comment:
let's put lower default, something like 10 and 50

Reply:
I put 15 - 30. Wdyt? Just so we can easily cover the initial amount of 11 categories we have.

Reply:
sure